source/common/tracing.js

```diff
@@ -62,19 +56,15 @@ module.exports = (context, serviceName) => {
       };
     }
 
-    // Enable everything that's available for auto-instrumentation
-    // if this is the server.
-    if (context === 'server') {
-      registerInstrumentations({
-        instrumentations: [getNodeAutoInstrumentations()],
-      });
-    }
+    registerInstrumentations({
+      instrumentations: [getNodeAutoInstrumentations()],
+    });
```

Comment:
Auto instrumentation will also load instrumentation for amqplib, so I just opted to enable this for every service.